client/my-sites/sharing/connections/connection.jsx

```diff
@@ -126,8 +126,7 @@ class SharingConnection extends Component {
 	}
 
 	getConnectionKeyringUserLabel() {
-		const { connection, siteId, translate, userId } = this.props;
```

Comment:
This line should probably remain, otherwise `userId` and `translate` won't be set. I'd suggest:

```js
const { connection, getUser, translate, userId } = this.props;
const keyringUser = getUser( connection.keyring_connection_user_ID );
```